### PR TITLE
More test retriers

### PR DESCRIPTION
**Pull request by noisekit**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 12,
   },
-  ignorePatterns: ['/*.js'],
+  ignorePatterns: ['!.*', '!.circleci/test-batch.js'],
```
Comment (0xjocke) [inline .eslintrc.js:18]: Is this right?  I'm reading it as "not ignoring all files in this folder and not ignoring `circleci/test-batch"`. Friday afternoon and double negatives, maybe I'm missing something?

Comment (noisekit) [reply .eslintrc.js:18]: yeah not ignore all of them and not ignore the one specific... idk, eslint is weird

Comment (noisekit) [reply .eslintrc.js:18]: We need to switch to biome anyway

Comment (0xjocke) [reply .eslintrc.js:18]: haha fair enough. I would've thought files were included by default.. But I'm sure there's reasons... Just wanted to confim it wasn't a mistake

**Review by 0xjocke — APPROVED** [review]

**Pull request merged**